deploy/cephfs/kubernetes/v1.13/helm/ceph-csi-cephfs/templates/nodeplugin-service.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ include "ceph-csi-cephfs.nodeplugin.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: csi-metrics
+    chart: {{ include "ceph-csi-cephfs.chart" . }}
+    component: {{ .Values.nodeplugin.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    app: {{ include "ceph-csi-cephfs.name" . }}
+    component: {{ .Values.nodeplugin.name }}
+    release: {{ .Release.Name }}
+  ports:
+    - name: http-metrics
+      port: 8080
+      protocol: TCP
+      targetPort: 8081
+    - name: grpc-metrics
+      port: 8090
+      protocol: TCP
+      targetPort: 8091
```

Comment:
Just a suggestion, we have these ports hardcoded, may be we can get it from `.Values`

Reply:
yes, @humblec having plan to move all configuration to Value.yaml as a follow-up PR.

Reply:
I have moved the ports for the current liveness sidecar to values.yaml in #570 
Wouldn't be to hard to rebase and implement this for you :)